beastling/util/misc.py

```diff
@@ -1,8 +1,38 @@
+from urllib.request import FancyURLopener
+
 import newick
 
 from beastling.util import log
 
 
+class FromOptions(object):
```

Comment:
Would this not be a good candidate for a `dataclass` (or `attr.s` if we prefer that – I don't know the pros and cons of either, dataclass is built-in, what else)?

Reply:
Oh, right, dataclass is new in 3.7, so in the interest of compatibility, attr.s makes more sense.